app/importers/daily_calls/twilio/retriever.rb

```diff
@@ -0,0 +1,31 @@
+require 'twilio-ruby'
+
+module DailyCalls
+  module Twilio
+    module Retriever
+      module_function
+
+      def from_api(start_date:, end_date:)
+        client = ::Twilio::REST::Client.new(account_sid, auth_token)
+
+        calls = []
+        page = client.calls.list('start_time>': start_date, 'start_time<': end_date + 1)
+
+        while page.any?
+          calls += page
+          page = page.next_page
+        end
+
+        calls
+      end
+
+      def account_sid
```

Comment:
These are a good example of things that feel opaque, and might become more transparent a dependency if they were pushed in during initialisation.